### PR TITLE
blog: announce 24.05 release

### PR DESCRIPTION
**Pull request by wegank**

### CODE REVIEW

File: src/content/blog/announcements/2024-05-31_nixos-24.05.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+GNOME has been updated to version 46 "Kathmandu", which introduces an enhanced Files app, support for OneDrive in Online Accounts, and more changes. Refer to the [release notes](https://release.gnome.org/46/) for more details.
+
+### Plasma 6
+
+Plasma has been updated to version 6.0, which introduces new defaults, refreshed Breeze, and more changes. Refer to the [release notes](https://kde.org/announcements/megarelease/6/) for more details.
+
+### Special Thanks
+
+We want to personally thank [Jacek Generowicz](https://github.com/jacg) for editorializing the release notes, [Vladimír Čunát](https://github.com/vcunat) for his tireless effort in managing jobsets, staging cycles and build infrastructure, [Cole Helbling](https://github.com/cole-h) for his epic effort in bisecting kernel issues with ARM64 for our remote builders, and [Graham Christensen](https://github.com/grahamc) for dutifully tending to our build infrastructure.
```
Comment (Mic92) [inline src/content/blog/announcements/2024-05-31_nixos-24.05.md:37]: Does Graham still manage the NixOS build infrastructure? I think Hexa is doing now a fair part there. I don't know who else is involved.

File: src/content/blog/announcements/2024-05-31_nixos-24.05.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
```
Comment (Mic92) [inline src/content/blog/announcements/2024-05-31_nixos-24.05.md:38]: - [Lennart Mühlenmeier](https://github.com/riotbib) For doing the artwork.

File: src/content/blog/announcements/2024-05-31_nixos-24.05.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+Plasma has been updated to version 6.0, which introduces new defaults, refreshed Breeze, and more changes. Refer to the [release notes](https://kde.org/announcements/megarelease/6/) for more details.
+
+### Special Thanks
+
+We want to personally thank [Jacek Generowicz](https://github.com/jacg) for editorializing the release notes, [Lennart Mühlenmeier](https://github.com/riotbib) for doing the release logo, [Vladimír Čunát](https://github.com/vcunat) for his tireless effort in managing jobsets and staging cycles, [Martin Weinelt](https://github.com/mweinelt) for dutifully tending to our build infrastructure, and [Cole Helbling](https://github.com/cole-h) for his epic effort in bisecting kernel issues with ARM64 for our remote builders.
+
+### Reflections and Closing
+
+I am very thankful for this opportunity to learn about and participate in the release process. It has been an awesome experience to work with everyone on this release, and I want to say thank you to everyone in this community that built NixOS and helped made this release better.
```
Comment (nat-418) [inline src/content/blog/announcements/2024-05-31_nixos-24.05.md:43]: Who is the "I" here?

Comment (wegank) [reply src/content/blog/announcements/2024-05-31_nixos-24.05.md]: Changed to "we".

**Review by thilobillerbeck — APPROVED** [review]

**Comment by thilobillerbeck** [comment]
What about this?

https://github.com/NixOS/nixos-homepage/pull/1436

**Comment by thilobillerbeck** [comment]
Stupid me

File: src/content/blog/announcements/2024-05-31_nixos-24.05.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+Plasma has been updated to version 6.0, which introduces new defaults, refreshed Breeze, and more changes. Refer to the [release notes](https://kde.org/announcements/megarelease/6/) for more details.
+
+### Linux Kernel 6.6
+
+The default kernel package has been updated from 6.1 to 6.6. All supported kernels since 4.19 remain available.
+
+### Special Thanks
+
+We want to personally thank [Jacek Generowicz](https://github.com/jacg) for editorializing the release notes, [Lennart Mühlenmeier](https://github.com/riotbib) for doing the release logo, [Vladimír Čunát](https://github.com/vcunat) for his tireless effort in managing jobsets and staging cycles, [Martin Weinelt](https://github.com/mweinelt) for dutifully tending to our build infrastructure, and [Cole Helbling](https://github.com/cole-h) for his epic effort in bisecting kernel issues with ARM64 for our remote builders.
```
Comment (Mic92) [inline src/content/blog/announcements/2024-05-31_nixos-24.05.md:43]:
```suggestion
We want to personally thank [Jacek Generowicz](https://github.com/jacg) for editorializing the release notes, [Lennart Mühlenmeier](https://github.com/riotbib) for doing the release logo, [Vladimír Čunát](https://github.com/vcunat) for his tireless effort in managing jobsets and staging cycles, [Martin Weinelt](https://github.com/mweinelt) for dutifully tending to our build infrastructure, and [Cole Helbling](https://github.com/cole-h) for his effort on keeping ofBorg running.
```

**Pull request merged**